StatisticService.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+
+struct GameRecord: Codable {
+    let correct: Int
+    let total: Int
+    let date: Date
+    func isNewRecord(current: Int) -> Bool {
+        return current > self.correct
+    }
+}
+protocol StatisticService {
+    var totalAccuracy: Double { get }
+    var gamesCount: Int { get }
+    var bestGame: GameRecord { get }
+    func store(correct: Int, total: Int)
+}
+
+final class StatisticServiceImplementation: StatisticService {
+    private let userDefaults = UserDefaults.standard
+    private enum Keys: String {
+        case correct, total, bestGame, gamesCount, totalAccuracy
+    }
+    private (set) var bestGame: GameRecord {
```

Comment:
Предпочтительна вот такая запись private(set) 
```suggestion
    private(set) var bestGame: GameRecord {
```